app/src/main/kotlin/com/exallium/todoapp/editnote/EditNoteViewImpl.kt

```diff
@@ -91,4 +91,8 @@ class EditNoteViewImpl(val bundle: Bundle) : BaseViewImpl<EditNoteView, EditNote
     override fun toggleSubmit(enable: Boolean) {
         saveNote.isEnabled = enable
     }
+
+    override fun showAllNotes(args: Bundle) {
+        router.popCurrentController()
```

Comment:
changed to router.pushController(RouterTransaction.with(AllNotesViewImpl(args)))